src/community/schemaless-features/mongodb-schemaless/src/main/java/org/geoserver/schemalessfeatures/mongodb/data/MongoSchemalessFeatureSource.java

```diff
@@ -219,19 +218,6 @@ protected void visitBinaryComparisonOperator(BinaryComparisonOperator filter) {
         return new Filter[] {splitter.getFilterPre(), splitter.getFilterPost()};
     }
 
-    private Map<String, String> getIndexesInfoMap() {
```

Comment:
Removed because $near operator required spatial index, but $geoIntersects does not have such requirement

Reply:
Reverted this